whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/InetnumSpec.groovy

```diff
@@ -6191,4 +6191,67 @@ class InetnumSpec extends BaseQueryUpdateSpec {
         then:
         created =~ /Create FAILED: \[inetnum] 192.168.0.0 - 192.168.0.255/
     }
+
+    def "Child creation breaks the hierarchy then error"() {
+        given:
+        databaseHelper.addObject("""\
+                    inetnum:    213.131.110.0 - 213.131.111.255
+                    netname:    RIPE-NCC
+                    status:     ASSIGNED PA
+                    descr:      description
+                    country:    NL
+                    admin-c:    TP1-TEST
+                    tech-c:     TP1-TEST
+                    mnt-by:     LIR-MNT
+                    source:     TEST
+                    """.stripIndent(true))
+        whoisFixture.reloadTrees()
```

Comment:
Use syncupdates rather than calling `reloadTrees()` (see the other tests don't need it)

Reply:
Done